modules/Bio/Otter/Lace/Client.pm

```diff
@@ -1291,7 +1293,7 @@ sub designate_this {
 sub get_slice_DE {
     my ($self, $slice) = @_;
     my $resp = $self->otter_response_content
-      ('GET', 'DE_region', { $self->slice_query($slice) });
+      ('GET', 'DE_region', { $self->slice_query($slice), 'author' => $self->author });
```

Comment:
Different formatting of the same construction in one file - somewhere "GET" is on the same line, and below - on the separate line, makes reading of the code worse

Reply:
I'm using the same formatting as that used in the file.